rs/verifier/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+use std::{str, time::Duration};
```

Comment:
@heeckhau i got verifier working in nodejs using a tool called [Neon](https://neon-bindings.com/) - will port over to tlsn-js